### DISCUSSION TITLE
SchemaError after upgrade to open-webui-0.3.21

### DISCUSSION THREAD

**Discussion by luochen1990** [open-webui/open-webui#5472] [General]
```
SchemaError: Invalid Schema:
 model.config.extra_fields_behavior
   Input should be 'allow', 'forbid' or 'ignore'
```

**Comment by mannp** [comment]
I have the same error with a fresh installation using open-webui-0.3.21.

**Comment by adamcstephens** [comment]
I have the same error when attempting to upgrade from 0.3.19 -> 0.3.21

Here's the full schema migration output: https://gist.github.com/adamcstephens/91ea2a7e713f39053dc036c87625a306

**Comment by drupol** [comment]
Same issue, I added `PYDANTIC_SKIP_VALIDATING_CORE_SCHEMAS="True";` as a temporary workaround to fix it.

**Comment by drupol** [comment]
This was also reported (and fixed) here: https://github.com/open-webui/open-webui/discussions/5506

But since we cannot upgrade to v0.3.22 on NixOS, this will need to wait a bit.

**Comment by Ag1O1** [comment]
@drupol where do i add `PYDANTIC_SKIP_VALIDATING_CORE_SCHEMAS="True";` ?

**Reply by drupol** [reply]
As an environment variable, before running `open-webui serve`.
